templates/sso73-x509-https.json

```diff
@@ -369,6 +388,12 @@
                                 "secret": {
                                     "secretName": "sso-x509-jgroups-secret"
                                 }
+                            },
+                            {
+                                "name": "service-ca",
+                                "configMap": {
+                                    "configMapName": "${APPLICATION_NAME}-service-ca"
```

Comment:
Replace `"configMapName"` with `"name"` like in the [RH-SSO 7.4 equivalent](https://github.com/jboss-container-images/redhat-sso-7-openshift-image/pull/145/files#r429224441) of the PR.

Current version returns
```
    error: DeploymentConfig.apps.openshift.io "sso" is invalid: [spec.template.spec.volumes[2].configMap.name: Required value, spec.template.spec.containers[0].volumeMounts[2].name: Not found: "service-ca"]
--> Failed
```
error. After the change the app is deployed from template successfully

Reply:
Done